### PR TITLE
Adding of Alga

### PR DESCRIPTION
**Pull request by thisiswhereitype**
Addition of `toGraph` function from alga library.

### CODE REVIEW

File: src/Pangraph/AlgebraicGraphs.hs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+module Pangraph.AlgebraicGraphs
```
Comment (snowleopard) [inline src/Pangraph/AlgebraicGraphs.hs]: There is no need for this module. Users can simply use `toGraph` to convert `Pangraph` values to any Alga graph, including `Graph a`.

File: src/Pangraph/GraphML/Parser.hs [hunk 2]
```diff
@@ -15,6 +15,10 @@ import qualified Pangraph.Internal.XMLTemplate       as PT
 parse :: BS.ByteString -> Maybe P.Pangraph
 parse file = either (const Nothing) (PT.hexmlToPangraph PT.graphMLTemplate) (H.parse file)
 
+<<<<<<< cdc68fb8f114a7d0591c6e4acf06735e91064be8
+-- | Like 'parse' except it throws an error on Nothing, which is when parsing fails.
+=======
```
Comment (snowleopard) [inline src/Pangraph/GraphML/Parser.hs]: Some kind of merge conflict here.

**Comment by thisiswhereitype** [comment]
I think I misunderstood `git merge`. Please review.

**Comment by snowleopard** [comment]
@thisiswhereitype Thanks! Looks good, so I merged.

**Pull request merged**